pkg/pipelines/bootstrap.go

```diff
@@ -46,11 +44,7 @@ const (
 	rolesPath             = "02-rolebindings/pipeline-service-role.yaml"
 	rolebindingsPath      = "02-rolebindings/pipeline-service-rolebinding.yaml"
 	serviceAccountPath    = "02-rolebindings/pipeline-service-account.yaml"
-	sealedSecretRolePath  = "02-rolebindings/sealed-secrets-aggregate-to-admin.yaml"
 	argocdAdminRolePath   = "02-rolebindings/argocd-admin.yaml"
-	secretsPath           = "03-secrets/gitops-webhook-secret.yaml" //nolint:gosec
-	authTokenPath         = "03-secrets/git-host-access-token.yaml" // nolint:gosec
-	dockerConfigPath      = "03-secrets/docker-config.yaml"
 	gitopsTasksPath       = "04-tasks/deploy-from-source-task.yaml"
```

Comment:
Do we want to renumber this files as well?

Reply:
+1 too.  Also check the other parts of the docs/readme files. eg. in Day 2 operations:

config/cicd/base/08-eventlisteners/cicd-event-listener.yaml